src/data/examples/index.ts

```diff
@@ -4,13 +4,13 @@ export const DEFAULT_EXAMPLE_LANGUAGES = ['javascript', 'react'];
 
 export const examples: Example[] = [
   {
-    id: 'chat-online-status',
-    name: 'Online Status',
+    id: 'chat-presence',
+    name: 'Presence',
```

Comment:
This might need to be `Chat presence` otherwise it will conflict with the Pub/Sub version